README.md

```diff
@@ -1 +1,44 @@
-# Webpack
\ No newline at end of file
+# To-Do List
+-This is an Awesome Booklist project that we did with my coding partner. The project allows the user to add or remove a book from the list.
+
+## Built With
+
+- Major languages 
+  HTML | Javascript
+- Frameworks
+  None
+  
+ ### Prerequisites
+HTML
+
+### Getting Started
+To get a local copy for practice or as a template, follow these simple steps.
+
+Access the live source on -https://github.com/sharon-odhiambo/TO-DO-LIST
+
+Copy the **https** or **SSH** address on the code
+Clone the repository using your editor   
```

Comment:
- [ ] [OPTIONAL] Your project works fine and that's great 👏 🔥. However, not everyone is familiar with using the projects that use NPM and Webpack. That means you should give that information here in the README file. Please show that your project uses webpack and someone should have NODE.js installed as a prerequisite. Another thing is that It would be better to add the steps to run your project. As you know someone has to run `npm install` to install all the project dependencies. In order to start the dev server, you know we have to run the command `npm start`. Please indicate all this information here in this section as they will make your README more informative😃 👍